### PR TITLE
Changed on-click behaviors + cleared out unused code.

### PR DESCRIPTION
**Pull request by kailbal**
Instead of opening whenever the path was changed from the default, the Game button shows up whenever 'Games' is clicked. Don't have anything specific, but figure it'd be a good start once we actually know what we want on the navigation bar.

### CODE REVIEW

File: frontend/src/components/NavMenu.vue [hunk 1]
```diff
@@ -11,10 +11,10 @@
     <v-list nav>
       <v-list-item to="/">Home</v-list-item>
       <v-list-item to="/about">About</v-list-item>
-      <v-list-item v-if="appBarDisplay" :to="`/info/${defaultGame}`">Game</v-list-item>
+      <v-list-item @click="gameSelected = !gameSelected"> Games </v-list-item>
+      <v-list-item v-if="gameSelected" :to="`/info/${defaultGame}`">Game</v-list-item>
```
Comment (lynnbridge) [inline frontend/src/components/NavMenu.vue]: Add a v-divider before the game to add some separation between the games selected. Make sure it also only shows up with the gameSelected check.

**Comment by kailbal** [comment]
reduced Navigation Bar assets to what is in the mock up, so now it's just the three buttons.

File: frontend/src/components/NavMenu.vue [hunk 2]
```diff
@@ -8,13 +8,15 @@
         </v-icon>
       </v-btn>
     </v-toolbar>
+    
     <v-list nav>
       <v-list-item to="/">Home</v-list-item>
       <v-list-item to="/about">About</v-list-item>
-      <v-list-item v-if="appBarDisplay" :to="`/info/${defaultGame}`">Game</v-list-item>
+      <!-- Game List: add route that goes to all games page-->
+      <v-list-item> Game List </v-list-item>
```
Comment (lynnbridge) [inline frontend/src/components/NavMenu.vue]: add `to="/games"

**Review by lynnbridge — APPROVED** [review]
Thanks for making those changes.

**Pull request merged**